src/main/steps/applicant2Sequence.ts

```diff
@@ -71,6 +73,11 @@ const sequences: Step[] = [
     getNextStep: data =>
       data.applicant2ApplyForFinancialOrder === YesOrNo.YES ? APPLY_FINANCIAL_ORDER_DETAILS : CHECK_ANSWERS_URL,
   },
+  {
+    url: CHECK_JOINT_APPLICATION,
+    getNextStep: data =>
+      data.applicant2Confirmation === YesOrNo.YES ? CHECK_ANSWERS_URL : SENT_TO_APPLICANT2_FOR_REVIEW,
```

Comment:
I have the next step for this page set up in this PR (https://github.com/hmcts/nfdiv-frontend/pull/826). We can link it together at a later date though.